app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt

```diff
@@ -38,3 +41,20 @@ fun TextView.clearFocusOnDone() {
         }
     }
 }
+
+fun View.clipWithRoundedCorners(curvedRadio: Int = 16.dp) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        outlineProvider = object : ViewOutlineProvider() {
+            override fun getOutline(view: View, outline: Outline) {
```

Comment:
🔝 Didn't knew about this. nice!!!

Reply:
https://youtu.be/0p_1QSUsbsM?t=10

Reply:
And here I thought it was a video talking about the ViewOutlineProvider!  🤣 🤣 🤣